### PR TITLE
Potential fix for disconnection

### PR DESCRIPTION
**Pull request by albaintor**
I have not tested it yet : I just added try/catch clauses to return a status code for each method, and most of all I let the code continue if the turn_on does not raise any exception.
I will try to test it on my own but first I have to configure a docker image...

### CODE REVIEW

**Review by zehnm — CHANGES_REQUESTED** [review]
I've included the turn_on workaround in my patch release fix (PR #17). Therefore I suggest to close this PR.

This doesn't fix the invalid entity state which causes this issue. I will start working on that next week.
An invalid entity state also causes issue in the UI and other logic.

File: .idea/integration-appletv.iml
[the remark this answers is not in the document]
Comment (albaintor) [reply .idea/integration-appletv.iml:1]: done

**Comment by albaintor** [comment]
Done I have committed and merged things, I'll let you review it and if it is okay I'll close the request

**Comment by albaintor** [comment]
Is there a possibility for me to test it ? you have beta releases somewhere ?
I didn't manage to make the integration work with a docker VM (running but not detected by the remote)

**Comment by zehnm** [comment]
Closing this PR, workaround is included in 0.13.4

You can test it running on a PC and register it as an additional driver with the REST-API if auto-discovery doesn't work.
Make sure that you change the driver_id and name.

`POST /api/intg/drivers`
```json
{
	"driver_id": "uc_appletv_driver2",
    "driver_url": "ws://MYIP:9090",
	"version": "0.13.4",
	"min_core_api": "0.7.0",
	"name": { "en": "Apple TV2" },
	"icon": "custom:appletv.png",
	"description": {
		"en": "Control your Apple TV with Remote Two."
	},
	"developer": {
		"name": "Unfolded Circle ApS",
		"email": "hello@unfoldedcircle.com",
		"url": "https://www.unfoldedcircle.com"
	},
	"home_page": "https://www.unfoldedcircle.com",
	"setup_data_schema": {
		"title": {
			"en": "Integration setup"
		},
		"settings": [
			{
				"id": "info",
				"label": {
					"en": "Setup process"
				},
				"field": {
					"label": {
						"value": {
							"en": "..."
						}
					}
				}
			}
		]
	},
	"release_date": "2023-07-23"
}
```

**Pull request closed without merging**